### DISCUSSION TITLE
Difficult recovery after Internet failure

### DISCUSSION THREAD

**Discussion by drousset** [mastodon/mastodon#27008] [Q&A]
Hi,
I'm running my own mastodon server at home (4.1.7 with Yunohost). I experimented an internet failure during approximately 10 days, resulting in no connection at all. After recovering a correct connection, everything seems to be ok but I receive only a few posts per day from 3 or 4 accounts. 

Has anyone any idea of what happens and how to fix that ?
Thanks a lot
D.

**Comment by ryanmio** [comment]
Your Mastodon instance probably lost federation context with other servers during the outage. Can you inspect your server logs for issues related to remote servers not being reachable?
